### ISSUE TITLE
How to train conformer architecture for speaker recognition task, by changing the decoder and loss function. 

### ISSUE THREAD

**Issue by mujhenahiata** [NVIDIA/NeMo#7042]
I had a SSL unsupervised pretrain conformer checkpoint. I wanted to use it for speaker recognition task. what changes i need to make in my config to do the same @titu1994 @meghmak13

**Comment by nithinraok** [comment]
Use conformer encoder from [here](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_transducer_bpe.yaml), and decoder and loss function from [here](https://github.com/NVIDIA/NeMo/blob/main/examples/speaker_tasks/recognition/conf/titanet-large.yaml)

**Comment by mujhenahiata** [comment]
Thank you for replying at the earliest. 
I am attaching a sample of the config file
can you verify what all parameters are necessary in order to run the experiment.
  encoder : feat_out: ?         |          decoder : feat_in: ?
i am also attaching a sample training script. please suggest suitable changes

**Comment by nithinraok** [comment]
Looking good. Try to play with Noam and Cosine scheduler to find right parameters for your setup

**Comment by mujhenahiata** [comment]
can you share how to set them in these cases where we use some different encoder and decoder 
 encoder : feat_out: ? | decoder : feat_in: ? also how to set these values?

**Comment by mujhenahiata** [comment]
> Use conformer encoder from [here](https://github.com/NVIDIA/NeMo/blob/main/examples/asr/conf/conformer/conformer_transducer_bpe.yaml), and decoder and loss function from [here](https://github.com/NVIDIA/NeMo/blob/main/examples/speaker_tasks/recognition/conf/titanet-large.yaml)

what should be the base config in this case?

**Comment by mujhenahiata** [comment]
@nithinraok much awaited response?

**Comment by nithinraok** [comment]
Base config is titanet, since we want data loading and model training for Speaker Embeddings.
